latch_cli/snakemake/workflow.py

```diff
@@ -453,39 +455,46 @@ def get_fn_code(
             snakefile = Path("{snakefile_path}")
 
             lp = LatchPersistence()
+            pkg_root = Path(".")
             """,
             1,
         )
 
         code_block += reindent(
             r"""
-            pkg_root = Path(".")
 
-            exec_id_hash = hashlib.sha1()
-            token = os.environ["FLYTE_INTERNAL_EXECUTION_ID"]
-            exec_id_hash.update(token.encode("utf-8"))
-            version = exec_id_hash.hexdigest()[:16]
-
-            jit_wf_version = os.environ["FLYTE_INTERNAL_TASK_VERSION"]
-            res = execute(
-                gql.gql('''
-                query executionCreatorsByToken($token: String!) {
-                    executionCreatorByToken(token: $token) {
-                        flytedbId
-                        info {
-                            displayName
-                        }
-                        accountInfoByCreatedBy {
-                            id
+            if dry_run:
```

Comment:
```suggestion
            if os.environ.get("LATCH_SNAKEMAKE_DRY_RUN"):
```

or something similar, just no overwriting parameters please